pom.xml

```diff
@@ -6,10 +6,12 @@
 
         <jackson.version>2.15.0</jackson.version>
         <log4j2.version>2.20.0</log4j2.version>
+        <logback.version>1.3.8</logback.version>
```

Comment:
Can we use 1.4.8?

Reply:
Nope, would need to bump to java11 for that